### DISCUSSION TITLE
Please help, why I don't check auto confirm order, but the order is auto confirmed.

### DISCUSSION THREAD

**Discussion by eric-makro** [saleor/saleor#10609] [Q&A] [answered]
Please help, why I don't check auto confirm order, but order that I created from draft is auto confirmed become unfulfilled.

**Comment by peelar** [comment, marked as answer]
Hello @eric-makro 👋 

If you believe this is unwanted behavior, please [follow the bug template](https://github.com/saleor/saleor/issues/new?assignees=&labels=bug%2Ctriage&template=bug_report.yaml&title=Bug%3A+), and provide steps to reproduce.

**Reply by eric-makro** [reply]
Sure,

What are you trying to achieve?
- I want to check how to confirm an order using dashboard

Steps to reproduce the problem
1. I use this order setting

<img width="1407" alt="image" src="https://user-images.githubusercontent.com/110385557/190299989-1880c40d-e416-483e-9ee3-9a50ac7a64ed.png">

2. I created draft order manually in dashboard

<img width="1436" alt="image" src="https://user-images.githubusercontent.com/110385557/190300282-38c65c0e-7135-4952-9c31-abb857d7d257.png">

3. I finalized the draft order and it become order directly without any confirmation or approval

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/110385557/190300392-d506fb88-bc62-4517-ae85-3f88a01a9dc5.png">

Could you please help review, did I miss any steps or order confirmation only appears when order is created from store front checkout ?

Thank you

**Reply by peelar** [reply]
Thank you @eric-makro. 

This is the expected behavior. Unconfirmed orders can only be created from the checkout. If an admin creates a draft, it is assumed the order is valid and should be confirmed.

Please see [the documentation](https://docs.saleor.io/docs/3.x/dashboard/orders#order-types).
